front/src/common/HelpModal/HelpModal.tsx

```diff
@@ -0,0 +1,52 @@
+import { useState, useEffect } from 'react';
+
+import { useTranslation } from 'react-i18next';
+import ReactMarkdown from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+
+import ModalBodySNCF from 'common/BootstrapSNCF/ModalSNCF/ModalBodySNCF';
+import ModalHeaderSNCF from 'common/BootstrapSNCF/ModalSNCF/ModalHeaderSNCF';
+
+const expectedFileFirstLine = '<!-- OSRD Help Markdown File -->';
+const defaultText = '# contact@osrd.fr';
+
+function HelpModal() {
```

Comment:
We try to declare our components with arrow functions (we should probably add this in osrd website)

```suggestion
const HelpModal = () => {
```